.github/workflows/ci.yml

```diff
@@ -127,9 +127,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest-16-cores]
+        os: [gh-runner-scrypto-ubuntu-jammy-16-cores]
     steps:
       - uses: actions/checkout@v3
+      - name: Setup cmake
```

Comment:
Do you think it would be possible to move those additional steps to the
`.github/actions/setup-env/action.yml`?
instead of adding them individually to each job?